src/libmongoc/src/mongoc/mongoc-util.c

```diff
@@ -878,8 +878,8 @@ _mongoc_simple_rand_uint32_t (void)
 
    /* Ensure *all* bits are random, as RAND_MAX is only required to be at least
     * 32767 (2^15). */
-   return (((uint32_t) rand () & 0x7FFFu) << 0u) | (((uint32_t) rand () & 0x7FFFu) << 15u) |
-          (((uint32_t) rand () & 0x0003u) << 30u);
+   return (((uint32_t) rand_r (&_mongoc_simple_rand_seed) & 0x7FFFu) << 0u) | (((uint32_t) rand_r (&_mongoc_simple_rand_seed) & 0x7FFFu) << 15u) |
```

Comment:
Suggest using `_mongoc_rand_simple` instead of `rand_r` to fix Windows builds:

```
mongoc-util.c(881): error C4013: 'rand_r' undefined
```

Reply:
Fixed